src/App.js

```diff
@@ -36,7 +37,7 @@ class App extends React.Component {
           <Header user={this.state.user} onLogout={this.logoutHandler} />
           <Switch>
             <Route exact path="/">
-              {/* TODO: if the user is logged in, render the `BestBooks` component, if they are not, render the `Login` component */}
+              <BestBooks />
             </Route>
             {/* TODO: add a route with a path of '/profile' that renders a `Profile` component */}
           </Switch>
```

Comment:
Really nice work so far!

See this TODO here: Use React Router to add ability for user to navigate between Profile and Home "pages"